lib/watcher/agent-trace.js

```diff
@@ -1,4 +1,4 @@
-var agent = require('../agent');
```

Comment:
tried that. that causes an error in worker at https://github.com/strongloop/strong-supervisor/blob/master/lib/watcher.js#L59
we cannot change worker since it is used by heart-beat generator.